packages/core/content-manager/server/src/history/services/history.ts

```diff
@@ -177,25 +264,124 @@ const createHistoryService = ({ strapi }: { strapi: Core.Strapi }) => {
         getLocaleDictionary(),
       ]);
 
-      const versionsWithMeta = results.map((version) => {
-        return {
-          ...version,
-          meta: {
-            unknownAttributes: getSchemaAttributesDiff(
-              version.schema,
-              strapi.getModel(params.contentType).attributes
-            ),
```

Comment:
Did you mean to remove this? I don't see it moved anywhere in the diff and we still need it.

Reply:
ah shit didn't mean to remove it, just move it. thx